mgmtfn/k8splugin/contivk8s/k8s_cni_test.go

```diff
@@ -162,6 +164,17 @@ func setupTestServer() {
 		logger.Infof("k8s test plugin closing %s", driverPath)
 	}()
 
+	// make sure the listener is ready before returning
+	for count := 0; count < 5; count++ {
```

Comment:
very minor: feel free to go ahead without any of this change
Should we use `go chan` instead of active waiting; however, this is a test case so not a concern